website/source/docs/platform/k8s/run.html.md

```diff
@@ -43,59 +46,122 @@ the [recommended security features](/docs/internals/security.html). Currently,
 elected and every node will have a running Consul agent.
 
-The defaults will install both server and client agents. To install
-only one or the other, see the
-[chart configuration values](/docs/platform/k8s/helm.html#configuration-values-).
+### Customizing Your Installation
+If you want to customize your installation,
+create a `config.yaml` file to override the default settings.
+You can learn what settings are available by running `helm inspect values ./consul-helm`
+or by reading the [Helm Chart Reference](/docs/platform/k8s/helm.html).
 
-### Viewing the Consul UI
+Once you've created your `config.yaml` file, run `helm install` with the `-f` flag:
+
+```bash
+$ helm install hashicorp ./consul-helm -f config.yaml
+```
+
+If you've already installed Consul and want to make changes, you'll need to run
+`helm upgrade`. See the [Upgrading Consul on Kubernetes](/docs/platform/k8s/run.html#upgrading-consul-on-kubernetes)
+section for more details.
+
+## Uninstalling Consul
+Consul can be uninstalled via the `helm delete` command:
+
+```bash
+$ helm delete hashicorp
+release "hashicorp" uninstalled
+```
+
+-> If using Helm 2, run `helm delete --purge hashicorp`
+
+After deleting the Helm release, you need to delete the PersistentVolumeClaims
+that store Consul's data. These are not deleted by Helm due to a [bug](https://github.com/helm/helm/issues/5156).
+To delete, run:
+
+```bash
+$ kubectl get pvc -l chart=consul-helm
+NAME                                   STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+data-default-hashicorp-consul-server-0   Bound    pvc-32cb296b-1213-11ea-b6f0-42010a8001db   10Gi       RWO            standard       17m
+data-default-hashicorp-consul-server-1   Bound    pvc-32d79919-1213-11ea-b6f0-42010a8001db   10Gi       RWO            standard       17m
+data-default-hashicorp-consul-server-2   Bound    pvc-331581ea-1213-11ea-b6f0-42010a8001db   10Gi       RWO            standard       17m
+
+$ kubectl delete pvc -l chart=consul-helm
+persistentvolumeclaim "data-default-hashicorp-consul-server-0" deleted
+persistentvolumeclaim "data-default-hashicorp-consul-server-1" deleted
+persistentvolumeclaim "data-default-hashicorp-consul-server-2" deleted
+```
+
+## Viewing the Consul UI
 
 The Consul UI is enabled by default when using the Helm chart.
-For security reasons, it isn't exposed via a Service by default so you must
-use `kubectl port-forward` to visit the UI. Once the port is forwarded as
-shown below, navigate your browser to `http://localhost:8500`.
+For security reasons, it isn't exposed via a LoadBalancer Service by default so you must
+use `kubectl port-forward` to visit the UI: 
 
 ```
-$ kubectl port-forward consul-server-0 8500:8500
+$ kubectl port-forward service/hashicorp-consul-server 8500:8500
 ...
 ```
 
-The UI can also be exposed via a Kubernetes Service. To do this, configure
-the [`ui.service` chart values](/docs/platform/k8s/helm.html#v-ui-service).
```

Comment:
I think this comment is still valuable. Did you think it's redundant?

Reply:
If they're not using ACLs then they're exposing the Consul HTTP API to the world and even with ACLs the anonymous token lets you read all nodes and services. So I thought it best not to encourage it.

Reply:
Not necessarily, this depends on the security groups and whether they use an internal or an external load balancer. Currently, we allow setting annotations for the UI service, and on AWS, for example, you can create an internal load balancer with an annotation.

Reply:
Okay I'll add it back.